docs/Node Operators/configuration-variables.md

```diff
@@ -421,392 +1081,289 @@ Setting this number higher will cause Chainlink to select higher gas prices.
+- Default: _none_
 
-Tells Chainlink to log all SQL statements made using the default logger.
+Should be set as the externally reachable IP address of the Chainlink node.
 
-## LOG_SQL_MIGRATIONS
+#### P2P_ANNOUNCE_PORT
 
-- Default: `"true"`
+- Default: _none_
 
-Tells Chainlink to log all SQL migrations made using the default logger.
+Should be set as the externally reachable port of the Chainlink node.
 
-## LOG_TO_DISK
+#### P2P_BOOTSTRAP_PEERS
 
-- Default: `"true"`
+- Default: _none_
 
-Enables or disables the node writing to the `$ROOT/log.jsonl` file.
+Default set of bootstrap peers.
 
-## MIN_INCOMING_CONFIRMATIONS
+#### P2P_LISTEN_IP
 
-- Default: `"3"`
+- Default: `"0.0.0.0"`
 
-The number of block confirmations to wait before kicking off a job run. Setting this to a lower value improves node response time at the expense of occasionally submitting duplicate transactions in the event of chain re-orgs (duplicate transactions are harmless but cost some eth).
+The default IP address to bind to.
 
-> ⚠️ NOTE
-> The lowest value allowed here is 1, since setting to 0 would imply that logs are processed from the mempool before they are even mined into a block, which isn't possible with Chainlink's current architecture.
+#### P2P_LISTEN_PORT
 
-## ETH_NONCE_AUTO_SYNC
+- Default: _none_
 
-- Default: `"true"`
+The port to listen on. If left blank, the node randomly selects a different port each time it boots. It is highly recommended to set this to a static value to avoid network instability.
 
-Enables/disables running the NonceSyncer on application start
+#### P2P_PEER_ID
 
-## ETH_FINALITY_DEPTH
+- Default: _none_
 
-- Default: _automatically set based on Chain ID, typically 50_
+The default peer ID to use for OCR jobs. If unspecified, uses the first available peer ID.
 
-The number of blocks after which an ethereum transaction is considered "final"
-BlocksConsideredFinal determines how deeply we look back to ensure that transactions are confirmed onto the longest chain
-There is not a large performance penalty to setting this relatively high (on the order of hundreds)
+### Networking Stack V2
 
-It is practically limited by the number of heads we store in the database and should be less than this with a comfortable margin.
-If a transaction is mined in a block more than this many blocks ago, and is reorged out, we will NOT retransmit this transaction and undefined behaviour can occur including gaps in the nonce sequence that require manual intervention to fix.
-Therefore, this number represents a number of blocks we consider large enough that no re-org this deep will ever feasibly happen.
+#### P2PV2_ANNOUNCE_ADDRESSES
 
-## ETH_HEAD_TRACKER_HISTORY_DEPTH
+- Default: _none_
 
-- Default: _automatically set based on Chain ID, typically 100_
+P2PV2AnnounceAddresses contains the addresses the peer will advertise on the network in host:port form as accepted by net.Dial. The addresses should be reachable by peers of interest.
```

Comment:
```
P2PV2AnnounceAddresses
```

There are several places in this document where it looks like the Go function name is inserted instead of the Environment Variable name. Node Operators won't interact with the functions directly in most cases. Correct? I can update these.

Reply:
I changed these to the environment variable version.